### PR TITLE
feat : wishlist

### PR DESCRIPTION
**Pull request by RaghavendraaReddy**
- need a review, please.
- [live Sandbox](https://r4vu6c.csb.app/wishlist/wishlist.html)

### CODE REVIEW

File: wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+.nav{
+    display: grid;
+    background-color: #9bb7d4;
```
Comment (haarsh24) [inline wishlist/wishlist.css:7]: Declare color in root and use.

File: wishlist/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    grid-column: 2/4;
+    grid-row: 1/2;
+    list-style-type: none;
+    display: flex;
+    
+}
+.nav-a{
+    padding: 2rem;
+    text-decoration: none;
+    color: black;
```
Comment (haarsh24) [inline wishlist/wishlist.css:25]: Same here . use same color code everywhere.

**Review by haarsh24 — APPROVED** [review]
suggested some minor changes. 
You can merge

**Review by Rajitha04 — APPROVED** [review]
Try to use relative (rem) units for better responsive pages and use variable colors, rest looks good

**Pull request merged**